pkg/browser/browser.go

```diff
@@ -30,7 +30,11 @@ func ForOS(goos, url string) *exec.Cmd {
 		r := strings.NewReplacer("&", "^&")
 		args = append(args, "/c", "start", r.Replace(url))
 	default:
-		exe = "xdg-open"
+		if findExe("xdg-open") {
+			exe = "xdg-open"
+		} else {
+			exe = "wslview"
```

Comment:
Could we only fall back to `wslview` if it actually exists?

If we are on legit Linux but `xdg-open` for some reason isn't present, I would much rather get an error message saying `unknown command: xdg-open` than `unknown command: wslview`, since the user is much more likely to be able to do something about the former and not much about the latter.

Reply:
Yup, good point.